src/views/ModelPlan/Status/__snapshots__/index.test.tsx.snap

```diff
@@ -197,3 +197,197 @@ exports[`Model Plan Status Update page > matches snapshot 1`] = `
   </main>
 </DocumentFragment>
 `;
+
+exports[`Model Plan Status Update page > matches snapshot 2`] = `
+<DocumentFragment>
+  <main
```

Comment:
Quite a few of these snapshots are taking additional snapshots instead of updating existing ones.  We should look into this to figure out why this is happening in all the tests.  May be addressed in the other PR as well but should check back